pkg/apis/options/alpha_options.go

```diff
@@ -0,0 +1,31 @@
+package options
+
+// AlphaOptions contains alpha structured configuration options.
+// Usage of these options allows users to access alpha features that are not
+// available as part of the primary configuration structure for OAuth2 Proxy.
+//
+// :::warning
+// The options within this structure are considered alpha.
+// They may change between releases without notice.
+// :::
```

Comment:
In case you're wondering why I've added this like this, once we start using the reference generator in #903, this warning get's rendered nicely as a warning as show in the image below

![image](https://user-images.githubusercontent.com/9232216/98988134-174bf580-251f-11eb-97c6-6f99551c0c01.png)